src/sentry/utils/pytest/stale_database_reads.py

```diff
@@ -143,7 +143,7 @@ def run_and_clear_commit_hooks(*args, **kwargs):
 
     old_apply_async = Task.apply_async
 
-    def apply_async(self, args=(), kwargs=(), countdown=None):
+    def apply_async(self, args=(), kwargs=(), countdown=None, **options):
```

Comment:
Would it make sense to remove `countdown` arg since that would be captured in **options.

Looks like you'll need to update `src/sentry/testutils/helpers/task_runner.py` as well since that does some patching on `apply_async`